src/tests/work-item/work-item-list/page-objects/work-item-list.page.js

```diff
@@ -457,8 +457,8 @@ class WorkItemListPage {
     return this.workItemKebabMoveToBottomButton (parentElement).click();
   }
   /**Iterations Page object model */
-  iterationAddButton  (){
-    return element(by.css('.pficon-add-circle-o.fa-icon'));
+  iterationAddButton (){
+    return element(by.id('add-iteration-icon'));
   }
   clickIterationCreateLabel  (){
     return element(by.id("add-iteration")).click();
```

Comment:
@mindreeper2420 the functional tests still fail. Please update this to add-iteration-icon.
After making this change, the tests passed for me.